cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -241,9 +241,14 @@ func (s *CMServer) Run(_ []string) error {
 	resourceQuotaController := resourcequotacontroller.NewResourceQuotaController(kubeClient)
 	resourceQuotaController.Run(s.ResourceQuotaSyncPeriod)
 
-	namespaceController := namespacecontroller.NewNamespaceController(kubeClient, s.NamespaceSyncPeriod)
+	namespaceController := namespacecontroller.NewNamespaceController(kubeClient, s.EnableHorizontalPodAutoscaler, s.NamespaceSyncPeriod)
 	namespaceController.Run()
```

Comment:
I'm not sure if we want it. There may be many other experimental controllers (deployments, deamons, ...). I think we should have a separate flag for enabling each of them, and take alternative of them here.